addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -122,7 +135,7 @@
     </t>
     <t t-elif="not channel.is_member">
         <div class="alert my-0 bg-200 text-center">
-            Course Not Buyable
+            Unavailable Course
```

Comment:
I think "Course unavailable", "Course currently unavailable", "Course unavailable for purchase" sound better, no? (so with "Course" first).